packages/test/src/constants/constants.ts

```
@@ -1,17 +1,50 @@
-import { WBTCArbitrumLocalnetAsset } from "@gardenfi/orderbook";
+// import { WBTCArbitrumLocalnetAsset } from '@gardenfi/orderbook';
 
-import { WBTCEthereumLocalnetAsset } from "@gardenfi/orderbook";
+// import {
+//   WBTCEthereumLocalnetAsset,
+//   bitcoinRegtestAsset,
+// } from '@gardenfi/orderbook';
 
-import { bitcoinRegtestAsset } from "@gardenfi/orderbook";
+import { SupportedAssets } from '@gardenfi/orderbook';
```

ESLint

[error] 'SupportedAssets' is defined but never used.

(@typescript-eslint/no-unused-vars)